roles/copy_additional_files/tasks/main.yml

```diff
@@ -33,3 +44,5 @@
   with_items:
     - 413
     - 500
+    - 502
+  when: ansible_distribution_version == '16.04'
```

Comment:
The when is executed for every item, so you could compact this to

```
 - name: Place nginx error files
    copy:
     src: /usr/share/nginx/html/index.html
     dest: "{{ galaxy_errordocs_dest }}/{{ item }}/index.html"
     remote_src: true
    with_items:
      - 413
      - 500
      - 502
    when: (item == 502 and ansible_distribution_version == '16.04')
```

if the idea is to skip 502 for 14.04. I haven't tested it though. If that doesn't work it looks OK to me.